.github/actions/fuse_compat/action.yml

```diff
@@ -24,7 +24,10 @@ runs:
           bash ./tests/fuse-compat/test-fuse-compat.sh 1.0.56 base
           bash ./tests/fuse-compat/test-fuse-compat.sh 1.0.56 01_meta_compression 01_flashback
           bash ./tests/fuse-compat/test-fuse-compat.sh 1.1.30 base
-
+          bash ./tests/fuse-compat/test-fuse-compat.sh 1.1.30 02_meta_compression_v3_to_v4 02_flashback_v3_to_v4
```

Comment:
if the ci passed, maybe we could remove some old compact tests later.

Reply:
unfortunately,  the compatibility *in general*, is not transitive (yet).

if C is compatible with B, and B compatible with A; C may be incompatible with A.

e.g. if in version C, the types that are supposed be to sealed in A are modified by accident, in the compact test, we only generate data of version B, and test C against version B, we can not detect that C can no longer read data of version A correctly. (since the test will not cover case that migrating from C to A).

this way of backward compatibility testing seems not sustainable though :thinking: